packages/login/network/BitcoinNode.js

```diff
@@ -7,14 +7,13 @@ import axios from 'axios'
 import { networks } from 'bitcoinjs-lib'
 import BitcoinBlockexplorerNode from './BitcoinBlockexplorerNode'
 import BitcoinMiddlewareNode from './BitcoinMiddlewareNode'
-import { BLOCKCHAIN_BITCOIN, BLOCKCHAIN_LITECOIN } from '../network/BitcoinProvider'
 
 const BTC_MAINNET_NODE = new BitcoinMiddlewareNode({
   api: axios.create({
     baseURL: 'https://middleware-bitcoin-mainnet-rest.chronobank.io',
     timeout: 4000,
   }),
-  blockchain: BLOCKCHAIN_BITCOIN,
+  blockchain: 'Bitcoin',
```

Comment:
Please add TODO comment like "// TODO: To replace hardcoded name back to const BLOCKCHAIN_BITCOIN when issue with imports will be resolved."

Same for the lines below.